tap_salesforce/__init__.py

```diff
@@ -1,23 +1,20 @@
 #!/usr/bin/env python3
-import json
 import sys
+from typing import Tuple, Optional, Dict
+from datetime import datetime, timezone, date
+
 import singer
 import singer.utils as singer_utils
-from singer import metadata, metrics
-import tap_salesforce.salesforce
-from tap_salesforce.sync import (
-    sync_stream,
-    resume_syncing_bulk_query,
-    get_stream_version,
-)
-from tap_salesforce.salesforce import Salesforce
-from tap_salesforce.salesforce.bulk import Bulk
-from tap_salesforce.salesforce.exceptions import (
+import requests
+
+from tap_salesforce.stream import Stream
+from tap_salesforce.client import Salesforce, Field
+from tap_salesforce.exceptions import (
     TapSalesforceException,
     TapSalesforceQuotaExceededException,
-    TapSalesforceBulkAPIDisabledException,
 )
 
+
```

Comment:
line 24-25 `"api_type", "select_fields_by_default",`should be deleted.

Reply:
it would be great if you add
```if __name__=='__main__':
    main()
```
so that I don't need to add it every time I test. Thanks!

Reply:
haha, done <3